### DISCUSSION TITLE
Achievements in user profile

### DISCUSSION THREAD

**Discussion by mtch99** [community/community#46806] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I think it is a good idea to include achievements on user profiles. It would be even better if we all contribute to the achievement system design

**Comment by Faiqlatheef** [comment]
After answering two discussions, You will get Galaxy Brain badge
And Add 2 or more branches and create pull request and pull it You will get one
More details here:
https://github.com/Schweinepriester/github-profile-achievements

**Comment by github-staff** [comment]
Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community.

Note that this change does not affect your ability to earn badges in other projects and communities across GitHub. This change only applies to GitHub’s Community Discussions.

You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀
